src/Eval.cpp

```diff
@@ -195,8 +195,9 @@ void Eval::debug (bool value)
 std::vector <std::string> Eval::getOperators ()
 {
   std::vector <std::string> all;
-  for (unsigned int i = 0; i < NUM_OPERATORS; ++i)
-    all.push_back (operators[i].op);
+  all.reserve(sizeof(operators));
+  for (const auto &opr : operators)
+    all.push_back (opr.op);
```

Comment:
I'd suggest using a more descriptive name, i.e. `operator` as the loop variable.

Reply:
operator might be a reserved keyword. At least nvim highlighted it. I was hoping to avoid confusion.

Reply:
yeah just tried it. Doesn't work.